sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/RecognizeLinkedEntities.java

```diff
@@ -26,9 +26,10 @@ public static void main(String[] args) {
 
         client.recognizeLinkedEntities(document).forEach(linkedEntity -> {
             System.out.println("Linked Entities:");
-            System.out.printf("Name: %s, entity ID in data source: %s, URL: %s, data source: %s.%n",
+            System.out.printf("Name: %s, entity ID in data source: %s, URL: %s, data source: %s,"
+                    + " Bing Entity Search API ID: %s.%n",
```

Comment:
consider adding a check to verify there is content in the property and then print it

Reply:
hmm and actually, it might be better not to include it in the samples as this is a super specific type that will only be used by advanced customers... so better if we don't confuse our normal users